src/hotel.js

```diff
@@ -0,0 +1,53 @@
+class Hotel {
```

Comment:
There are three instances that I see are a bit repetitive, bookings on date(viewRoomsAvailable), bookings on date(returnTotalRevenue), and guest bookings(returnTotalSpent), These are all incredibly similar and I believe you could make these into a helper method and you would just substitute the date or userID in through an argument.  Then you would call on that helper method inside these methods and just pass the right param, it would honestly just make it look shorter so it's not a big deal.